src/Fixer/DeprecatedFixerInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer;
+
+/**
+ * @author SpacePossum
+ */
+interface DeprecatedFixerInterface
```

Comment:
not sure about this one, could be nice, but it's not used anywhere. 
as for 2.x line we want to not have deprecated fixers in the first place.

this could be nice when we will prepare for 3.x line, but maybe it's not needed for now?